WiseBuy/View Controllers/ComposeViewController.m

```diff
@@ -39,8 +47,8 @@ - (IBAction)didTapPost:(id)sender {
     
     [post saveInBackgroundWithBlock:^(BOOL succeeded, NSError * _Nullable error) {
         if (!succeeded) {
-            [AlertManager cannotPostDeal:self];
             NSLog(@"%@", error.description);
+            // TODO: Send an alert to user saying there's an error
```

Comment:
Is this an intentional revert?

Reply:
it was accidentally deleted so i reverted it

Reply:
i just took a look again and seems like it was accidentally deleted for this PR, i will add it back